alica_simple_solver/CMakeLists.txt

```diff
@@ -9,35 +9,92 @@ if (${CMAKE_EXTRA_GENERATOR} MATCHES "Eclipse CDT4")
 	add_definitions (-DCMAKE_ECLIPSE_GENERATE_SOURCE_PROJECT=TRUE)
 endif (${CMAKE_EXTRA_GENERATOR} MATCHES "Eclipse CDT4")
 
-find_package(catkin REQUIRED COMPONENTS alica_engine alica_common_config alica_solver_interface)
 
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES alica_simple_solver
-  CATKIN_DEPENDS alica_engine alica_common_config alica_solver_interface
-)
+# Set up utils to allow Colcon/catkin to find the packages
+include(GNUInstallDirs)
+include(CMakePackageConfigHelpers)
+set(ConfigPackageLocation ${CMAKE_INSTALL_LIBDIR}/${CMAKE_PROJECT_NAME}/cmake) 
+
+
+# Imports
+find_package(alica_engine REQUIRED)
+find_package(alica_common_config REQUIRED)
+find_package(alica_solver_interface REQUIRED)
+
 
+#Build
 include_directories(
- include
- ${catkin_INCLUDE_DIRS}
+  include
+  ${alica_common_config_INCLUDE_DIRS}
+  ${alica_solver_interface_INCLUDE_DIRS}
```

Comment:
Removed all the ones that it let me - since alica_common_config is just headers it seems like it still needs it in the directories since it has no compiled file to link against